rdmd_test.d

```diff
@@ -138,38 +138,40 @@ void runCompilerAgnosticTests(string rdmdApp, string defaultCompiler, string mod
 {
     /* Test help string output when no arguments passed. */
     auto res = execute([rdmdApp]);
-    assert(res.status == 1, res.output);
-    assert(res.output.canFind("Usage: rdmd [RDMD AND DMD OPTIONS]... program [PROGRAM OPTIONS]..."));
+    enforce(res.status == 1, res.output);
+    enforce(res.output.canFind("Usage: rdmd [RDMD AND DMD OPTIONS]... program [PROGRAM OPTIONS]..."));
 
     /* Test --help. */
     res = execute([rdmdApp, "--help"]);
-    assert(res.status == 0, res.output);
-    assert(res.output.canFind("Usage: rdmd [RDMD AND DMD OPTIONS]... program [PROGRAM OPTIONS]..."));
+    enforce(res.status == 0, res.output);
+    enforce(res.output.canFind("Usage: rdmd [RDMD AND DMD OPTIONS]... program [PROGRAM OPTIONS]..."));
 
     string helpText = res.output;
 
     // verify help text matches expected defaultCompiler
     {
+        version (Windows) helpText = helpText.replace("\r\n", "\n");
```

Comment:
This doesn't seem like a valid solution.  We're meant to be testing that the help text meets expectations: if we have different expectations on Windows than on other platforms, then the test check on the contents should be different, we shouldn't rewrite the contents of the generated help text to meet the test conditions expected on Posix.

Reply:
Kind of agreed but the test is about checking what compiler is in it, not what line endings it uses.